projects/plugins/jetpack/extensions/blocks/videopress/editor.js

```diff
@@ -132,19 +166,24 @@ const addVideoPressSupport = ( settings, name ) => {
 
 	const { deprecated, edit, save, supports, transforms } = settings;
 	const { available, unavailableReason } = getJetpackExtensionAvailability( 'videopress' );
+	const isNotAvailable =
+		( isSimpleSite() || isAtomicSite() ) &&
+		[ 'missing_plan', 'unknown' ].includes( unavailableReason );
+
+	const resumableUploadEnabled = typeof window.videoPressResumableEnabled !== 'undefined';
```

Comment:
See previous comment on videoPressResumableEnabled (maybe this could be in a function/method somewhere ?)